nss-cohort-36/bangazon-client-imagination-station-react
# Lr add payment type 7

# Description
User should be able to go to profile in Navbar and click the view payment types button and all payment types pop up. If you press delete, the payment type will be deleted. If you go to add new payment type and add a new one, it should show up in the list when you go to payment types list. 

Fixes #7 #8 

Please delete options that are not relevant.

# Testing Instructions
User should be able to go to profile in Navbar and click the view payment types button and all payment types pop up. If you press delete, the payment type will be deleted. If you go to add new payment type and add a new one, it should show up in the list when you go to payment types list.

# Checklist:
